pyproject.toml

```diff
@@ -43,6 +43,8 @@ dependencies = [
   'PyYAML>=6.0.0',
   'mmh3>=4.0.1',
   'orjson>=3.9.12',
+  'mlx>=0.0.5',
```

Comment:
Remove those and make them dynamic in `embedding_functions` with instructions on how the user who wants to use MLX needs to install the required dependencies.